src/handbook/product/verticals.md

```diff
@@ -4,14 +4,74 @@ navTitle: Market Verticals
 
 # Product Verticals
 
-## Background
-Node-RED is best used for connecting _things_ together, and doing logic or manipulation of the data in that connection. Those things can be wide ranging from Web APIs using HTTP to physical hardware on serial connections.
-FlowForge competes in specific market verticals which allow us to optimize for certain use-cases.
+Product verticals refer to specific, identifiable categories within an industry or market that have distinct characteristics and needs. By understanding the verticals, we can align our [product strategy](./strategy.md) to meet those specific needs. For this overview, we will focus on three distinct product verticals: Industrial IoT Platforms, Low-Code Tools, and Data Integration Tools.
 
+## Industrial IoT Platforms
 
-## Digital Integration 
-Also known as IPaaS (Integration Platform as a Service) This market serves business users looking to integrate workflows across multiple different systems. Typically these are Cloud Hosted SaaS types of systems, usually with some sort of Public API.
-There is also the scenario of integrating between cloud systems and internal company systems or data stores.
+An IIoT platform is a solution that solves specific problems and concrete use cases, as well as allows integration with an ecosystem of other technologies. Node-RED's ability to connect and process data from IIoT devices and close the gap between IT and OT aligns with this category. It is especially relevant in the manufacturing sector, where such data is vital for operational efficiency and innovation. FlowForge, by adding Enterprise features, aids in managing remote deployments and delivery pipelines, enhancing the overall capabilities of Node-RED as an IIoT platform.
+
+However, a potential downside is that it may currently be perceived as too general-purpose and not specialized enough for specific IIoT tasks compared to other, more narrowly focused IIoT platforms.
+
+### [Market Size](https://www.imarcgroup.com/industrial-iot-market)
+
+The Industrial IoT market is expected to reach a value of $516.6 Billion by 2028 ($221.7 Billion in 2022), with a compound annual growth rate (CAGR) of 15.49% from 2023 to 2028. About $66.5 Billion of this was in software alone in 2022.
+
+### Key Customers
+
+The primary customer base in the IIoT sector includes manufacturing, energy, transportation, and utilities companies. These businesses prioritize operational efficiency, equipment monitoring, predictive maintenance, and often operate on a global scale. They require a system capable of handling large data volumes and enabling swift, insightful decision-making.
+
+### Participants in the market
+
```

Comment:
Here is a link to the Garter MQ for IIOT Platform that includes the main players.  https://www.gartner.com/doc/reprints?id=1-2C757S9J&ct=230105&st=sb

Reply:
I like the definition of Gartner (will include it), I 100% disagree with Gartner on the "main players" and the results

Reply:
But of course, we can add more players. I just absolutely do not agree with their ranking, and I don't know how such an outcome can be generated.

Reply:
Interesting who do you consider to be the main players?

Reply:
I can agree with Gartner on:

- Microsoft
- AWS
- Siemens
- PTC
- Hitachi

But Siemens and Hitachi are definitely no visionaries, and why is AWS ranked as a challenger?

who is missing, players like:
- Tulip
- HighByte
- Ignition 

These are significant players, maybe there are in different categories?

Reply:
Ah, Tulip is in the MES ranking. That's an interesting choice, but it aligns with Tulip's marketing.

Reply:
HighByte is not a IIoT Platform. They are focused on the data ops.  HighByte is also not a major player.

Ignition was actually considered but not included in the report.

I would not consider Tulip an IIoT Platform. As you mentioned they are an MES and even then they don't want to be called an MES.